Revit_Engine/Create/Elements/BuildingElement.cs

```diff
@@ -43,8 +43,8 @@ public static partial class Create
         [Input("curve", "Bottom curve describing shape of wall BuilidngElement")]
         [Input("height", "Height of wall BuilidngElement")]
         [Input("familyTypeName", "Revit Family Type Name for wall BuilidngElement")]
-        [Output("BuildingElement")]
-        public static BuildingElement BuildingElement(ICurve curve, double height, string familyTypeName)
+        [Output("Environment Panel")]
+        public static Panel BuildingElement(ICurve curve, double height, string familyTypeName)
```

Comment:
Method shall be renamed from BuildingElement to Panel

Reply:
Revit_Engine/Create/Elements/BuildingElement.cs to be renamed to Revit_Engine/Create/Elements/Panel.cs